### PR TITLE
Add documentation for virtual networks UI

### PR DESCRIPTION
**Pull request by cbosdo**
# Description

Add virtualization > Networks tab documentation in the reference guide

# Target branches

To what product version this document applies?

- [ ] Manager-3.2
- [ ] Manager-4.0
- [x] Manager-4.1
- [X] Uyuni

### CODE REVIEW

**Comment by cbosdo** [comment]
Will concern 4.1 once the corresponding PRs are merged

**Comment by Loquacity** [comment]
If you want to hit Uyuni only, we'll need to conditionalise it. Otherwise, we can leave it unmerged until it's ready for 4.1 (to be honest, we probably won't be refreshing the docs in the next week or two anyway).

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 1]
```diff
@@ -287,3 +287,24 @@ Some pool types will not allow you to delete the volumes or the pool.
...
+[guimenu]``Autostart``::
+The network starts automatically when the virtual host boots.
+
+[guimenu]``Persistent``::
+The network will be kept after being stopped.
+
+[guimenu]``Bridge``::
+The bridge interface used by the network.
+
+Each network can be started, stopped or deleted using the corresponding button in the list.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc:309]:
```suggestion
Each network can be started, stopped, or deleted using the corresponding button in the list.
```

File: modules/reference/pages/systems/system-details/sd-virtualization.adoc [hunk 2]
```diff
@@ -287,3 +287,24 @@ Some pool types will not allow you to delete the volumes or the pool.
...
+Theses actions can also by applied to multiple networks by selection the networks in the list and using the corresponding button above the list.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details/sd-virtualization.adoc:310]:
```suggestion
Theses actions can also by applied to multiple networks by selecting the networks in the list and using the corresponding button above the list.
```

**Comment by cbosdo** [comment]
> If you want to hit Uyuni only, we'll need to conditionalise it. Otherwise, we can leave it unmerged until it's ready for 4.1 (to be honest, we probably won't be refreshing the docs in the next week or two anyway).

Then we'll just need to keep the PR opened until the corresponding Uyuni PRs are merged. Will be for 4.1, but I have no idea when that will be.

**Comment by cbosdo** [comment]
The corresponding code has been merged in both master and 4.1 branches.

**Pull request merged**